### DISCUSSION TITLE
Font used in Demo

### DISCUSSION THREAD

**Discussion by Devlyn-Nelson** [fdehau/tui-rs#650] [Q&A] [answered]
Nearly 2 years ago an issue [(356)](https://github.com/fdehau/tui-rs/issues/356) was raised about the results of using braille markers and how the selected font can make a big difference. Can you guys just post the name of the font you guys used so people do not need to struggle finding one?

**Comment by fdehau** [comment, marked as answer]
👋 The alt text of the demo gif has the answer: `Inconsolata font 12pt`. This gif is pretty old though and these day I'm using [Iosevka Term](https://github.com/be5invis/iosevka).

**Reply by Devlyn-Nelson** [reply]
Thanks a bunch, we use tui for a lot of testing software and i don't ever have time to find this sort of stuff due to it being a very low priority.
